testdata/BUILD

```diff
@@ -677,6 +677,24 @@ py3_image(
     main = "py3_image.py",
 )
 
+container_image(
+    name = "py3_image_base_with_custom_run_flags",
+    base = "@py3_image_base//image",
+    cmd = ["-V"],
+    docker_run_flags = "-i --rm --network=host -e ABC=ABC",
+    legacy_run_behavior = False,
```

Comment:
should we have a test that uses legacy_run_behavior=true ?

Reply:
Also, why is this necessary here (i.e., what happens if you just don't set it, as I see none of the examples currently use legacy_run_behavior)?

Reply:
The default value for legacy_run_behavior is True. docker_run_flags is only used when legacy_run_behavior is false.